docs/source/llms/tracing/tracing-schema.rst

```diff
@@ -331,3 +331,117 @@ documents from a vector store). The ``RETRIEVER`` span type has the following sc
+
+Chat Completion Spans
+^^^^^^^^^^^^^^^^^^^^^
+
+Spans of type ``CHAT_MODEL`` or ``LLM`` are used to represent interactions with a chat completions API
+(for example, OpenAI's `chat completions <https://platform.openai.com/docs/api-reference/chat/create>`_,
+or Anthropic's `messages <https://docs.anthropic.com/en/api/messages>`_ API). As providers can have
+different schemas for their API, there are no restrictions on the format of the span's inputs and
+outputs.
+
+However, it is still important to have a common schema in order to enable certain UI features (e.g. rich
+conversation display), and to make authoring evaluation functions easier. To support this, we specify some
+custom attributes for standardized chat messages and tool defintions:
+
+.. list-table::
+    :widths: 20 40 40
+    :header-rows: 1
+    :class: wrap-table
+
+    * - **Attribute Name**
+      - **Description**
+      - **Note**
+
+    * - **mlflow.chat.messages**
+      - This attribute represents the system/user/assistant messages involved in the
+        conversation with the chat model. It enables rich conversation rendering in the UI,
+        and will also be used in MLflow evaluation in the future. 
+        
+        The type should be ``List[`` :py:class:`RequestMessage <mlflow.types.chat.RequestMessage>` ``]``,
+        following the standardized chat message format from `MLflow AI Gateway <https://mlflow.org/docs/latest/llms/deployments/index.html>`_
+      - This attribute can be conveniently set using the :py:func:`mlflow.tracing.utils.set_span_chat_messages` function. This function
+        will throw a validation error if the data does not conform to the spec.
+    
+    * - **mlflow.chat.tools**
+      - This attribute represents the tools that were available for the chat model to call. In the OpenAI
+        context, this would be equivalent to the `tools <https://platform.openai.com/docs/api-reference/chat/create#chat-create-tools>`_ 
+        param in the Chat Completions API.
+
+        The type should be a list of valid tools. At the moment, the available tool types are:
+
+        * :py:class:`FunctionTool <mlflow.types.chat.FunctionTool>`
+        * :py:class:`UnityCatalogFunctionTool <mlflow.types.chat.UnityCatalogFunctionTool>`
+
+      - This attribute can be conveniently set using the :py:func:`mlflow.tracing.utils.set_span_chat_tools` function. This function
+        will throw a validation error if the data does not conform to the spec.
+
+Please refer to the example below for a quick demonstration of how to use the utility functions described above, as well as
+how to retrieve them using the :py:class:`span.get_attribute() <mlflow.entities.Span.get_attribute>` function:
+
+.. code-block:: python
+
+  import mlflow
+  from mlflow.entities.span import SpanType
+  from mlflow.tracing.constant import SpanAttributeKey
+  from mlflow.tracing.utils import set_span_chat_messages, set_span_chat_tools
```

Comment:
```suggestion
  from mlflow.tracing import set_span_chat_messages, set_span_chat_tools
```

Can we add `set_span_chat_messages` and `set_span_chat_tools` in `mlflow/tracing/__init__.py`?